draft-ietf-mmusic-sdp-bundle-negotiation.xml

```diff
@@ -438,11 +431,13 @@
       </t>
         <section anchor="sec-sdp-oa-cat" title="Mux Category Considerations" toc="default">
         <t>
-          When an offerer associates SDP attributes with a bundled "m=" line associated
-          with a shared address, IDENTICAL and TRANSPORT mux category SDP attributes
-          <xref target="I-D.ietf-mmusic-sdp-mux-attributes" /> are associated with the
-          "m=" line only if the "m=" line is also associated with the offerer BUNDLE-tag.
-          Otherwise the offerer MUST NOT associate such SDP attributes with the "m=" line.
+          When an offerer associates SDP attributes with a bundled "m=" line (including any
+          bundle-only "m=" line) associated with a shared address, IDENTICAL and TRANSPORT
+          mux category SDP attributes <xref target="I-D.ietf-mmusic-sdp-mux-attributes" />
+          are associated with the "m=" line only if the "m=" line is also associated with
+          the offerer BUNDLE-tag. Otherwise the offerer MUST NOT associated such SDP attributes

```

Comment:
Fixed.

I will also combine the offerer- and answerer paragraphs into a single paragraph, in oder to get rid of some of the repetition you mentioned.